### DISCUSSION TITLE
need help -> new to ical4j; first tries

### DISCUSSION THREAD

**Discussion by GuKKDevel** [ical4j/ical4j#542] [Q&A]
Hello, 
I am trying to use ical4j in an Eclipse-IDE. So I downloaded from Sourceforge the latest version 3.0.0 of ical4j, added slf4j and slf4j-simple from the maven repository, same with commons-lang3 and commons-collections4. Included them into my classpath.

I coded your example

> FileInputStream fin = new FileInputStream(inPfad + kalender[0] + ".ics");
> CalendarBuilder builder = new CalendarBuilder();
> Calendar calendar = builder.build(fin);`

Now I get 

> [main] INFO net.fortuna.ical4j.util.Configurator - ical4j.properties not found.
> Exception in thread "main" java.lang.NoClassDefFoundError: javax/cache/configuration/Configuration
> 	at net.fortuna.ical4j.model.TimeZoneLoader.cacheInit(TimeZoneLoader.java:275)
>	at net.fortuna.ical4j.model.TimeZoneLoader.<init>(TimeZoneLoader.java:81)
>	at net.fortuna.ical4j.model.TimeZoneRegistryImpl.<init>(TimeZoneRegistryImpl.java:125)
>	at net.fortuna.ical4j.model.TimeZoneRegistryImpl.<init>(TimeZoneRegistryImpl.java:116)
>	at net.fortuna.ical4j.model.DefaultTimeZoneRegistryFactory.createRegistry(DefaultTimeZoneRegistryFactory.java:48)
>	at net.fortuna.ical4j.data.CalendarBuilder.<init>(CalendarBuilder.java:105)
>	at Ausprobieren.<init>(Ausprobieren.java:43)
>	at Ausprobieren.main(Ausprobieren.java:69)

>Caused by: java.lang.ClassNotFoundException: javax.cache.configuration.Configuration
>	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
>	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
>	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
>	... 8 more

Don't know where the mistake is. can you help me?

Kind Regards
GuKKDevel (Karl-Heinz)

**Comment by benfortuna** [comment]
FYI releases are no longer maintained on SourceForge, so you are best to get the latest JARs from Maven Central currently:

https://search.maven.org/artifact/org.mnode.ical4j/ical4j/3.2.1/jar
